src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2922,6 +2885,21 @@ List<Method> getDeclaredPublicMethods(String name, Class<?>... parameterTypes) {
         return result;
     }
 
+    /**
+     * Returns the most specific {@code Method} object of this class, super class or
+     * interface that have the specified method name and parameter types.
+     *
+     * @param publicOnly true if only public methods are examined, otherwise all methods
+     * @param name the name of the method
+     * @param parameterTypes the parameter array
+     * @return the list of {@code Method} objects for the public methods of
+     *         this class matching the specified name and parameters
```

Comment:
The return description seems to be copied from getDeclaredPublicMethods, I think you can drop it as the method description is clear.

Reply:
```
 * @return the {@code Method} object for the method found from this class matching
 * the specified name and parameters, or null if not found
```